analytics/criteo.js

```diff
@@ -0,0 +1,109 @@
+//
+// Criteo tracking for auctions product feed and artworks product feed.
+//
+
+window.criteo_q = window.criteo_q || []
+var pathSplit = location.pathname.split("/")
+var userEmail = function() {
+  return sd.CURRENT_USER ? [sd.CURRENT_USER.email] : [];
+}()
+if (pathSplit[1] === "auctions") {
+  // Auctions events
+  window.criteo_q.push(
+    { event: "setAccount", account: sd.CRITEO_AUCTIONS_ACCOUNT_NUMBER },
+    { event: "setSiteType", type: "m" },
+    { event: "viewHome" }
+  )
+} else if (pathSplit[1] === "auction") {
```

Comment:
I think you should document these path splits a little better, it would be annoying to reconstruct them for someone else coming to this code. Thinking of something like:

```
           0                       1              2                 3
http://www.artsy.net/auctions/:auction_id/bid
```